src/OpenLoco/GameCommands/GameCommands.h

```diff
@@ -141,6 +149,8 @@ namespace OpenLoco::GameCommands
 
     struct VehiclePlacementArgs
     {
+        static constexpr auto command = GameCommand::vehiclePlace;
+
         VehiclePlacementArgs() = default;
         explicit VehiclePlacementArgs(const registers regs)
```

Comment:
all of these structs should take a `const registers&` validate and modify as part of this pr.